### DISCUSSION TITLE
horovod function

### DISCUSSION THREAD

**Discussion by lunasdejavu** [Lightning-AI/pytorch-lightning#12457] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I want to know which version of pytorch, horovod, CUDA,NCCL  and lightning should I install so that I can use the trainer(accelerator= horovod)
it is useless for me for pytorch 1.7 horovod 0.23 lightning 1.5.10 CUDA 10.2
NCCL 2708
,pytorch 1.10 CUDA11.3 horovod0.23 lightning 1.5.10 is useless either

**Comment by akihironitta** [comment]
Hi @lunasdejavu!

We're testing the horovod integration with the following env in our CI at the time of `pytorch_lightning==1.6.0` release:
```
horovod                       0.23.0
torch                         1.8.1
torchmetrics                  0.7.2
torchtext                     0.9.1
torchvision                   0.9.1
```

Its docker image is available at https://hub.docker.com/r/pytorchlightning/pytorch_lightning:
```
pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8@sha256:b75de74d4c7c820f442f246be8500c93f8b5797b84aa8531847e5fb317ed3dda
```

Could you give it try with the above setup?
